lib/main_containers.pm

```diff
@@ -149,14 +150,14 @@ sub load_container_tests {
             }
             else {
                 # Common openQA image tests
-                load_image_tests_podman() if (/podman/i);
-                load_image_tests_docker() if (/docker/i);
+                load_image_tests_podman($run_args) if (/podman/i);
```

Comment:
I don't get why we need to pass it to this method, which doesn't need it.

Reply:
The `load_image_tests_podman` subroutine calls `load_image_test` subroutine which again passes `run_args` parameter to `loadtest`.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14542#discussion_r831873707